docs/examples/qubit_spectroscopy_cz_ramsey.ipynb

```diff
@@ -73,8 +73,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# if you are on the QPU you can look at `qc.device.specs`\n",
```

Comment:
Wasn't really sure of the right replacement for this in pyQuil v3. @erichulburd Any thoughts?

Reply:
`qc.quantum_processor.to_compiler_isa()` will contain spec information for specific gates on nodes and edges. This will only be interesting for `QCSQuantumProcessor`.

We don't expose the `_isa` publicly on `QCSQuantumProcessor`, though it would be as simple as adding an `isa` property.

Reply:
OK, thanks for the info. I'll leave this as is then